examples/investigate_streams.rs

```diff
@@ -242,21 +262,25 @@ fn main() {
             lzma: "lzma (b)",
         },
     );
-    let print = |in_extension: BTreeMap<String, Compressions<u64>>| {
-        for (name, info) in in_extension {
-            print_column(&options, &name, &info);
-        }
-    };
 
     // Output `js` and `binjs` first.
-    if let Some(in_extension) = stats.remove("js") {
-        print(in_extension);
+    if let Some(mut in_extension) = stats.remove("js") {
+        print_column(&options, "js", &in_extension.remove("js").unwrap());
     }
-    if let Some(in_extension) = stats.remove("binjs") {
-        print(in_extension);
+    if let Some(mut in_extension) = stats.remove("binjs") {
+        print_column(&options, "binjs", &in_extension.remove("binjs").unwrap());
     }
-    // Output the rest by alphabetical order.
-    for (_, in_extension) in stats {
-        print(in_extension);
+    println!("");
+
+    // Output the rest by `ranking` order.
+    let all_values = Itertools::flatten(stats.into_iter().map(|(_, value)| value.into_iter()));
```

Comment:
`.flatten()` is part of stdlib these days, so itertools should be unnecessary here.

Reply:
Actually, instead of even `.flatten()` it seems better to replace `.map(...)` with `.flat_map(...)` here.

Reply:
The problem here is that itertools' `flatten` and stdlib's conflict, so I need to desambiguate.

Reply:
What I'm suggesting is specifically to use stdlib's `.flatten()` (or, rather, `flat_map()`).